### DISCUSSION TITLE
Increasing the number of message passing steps

### DISCUSSION THREAD

**Discussion by zakmachachi** [ACEsuit/mace#75] [Q&A] [answered]
Hi!

I have a small question regarding the MACE input file. In this [paper](https://arxiv.org/pdf/2205.06643.pdf), there is discussion on increasing _t_ which refers to the message passing steps (end of page 9). I am interested in using a small cutoff but having a larger _t_ and seeing where the optimal _t_ value is for my system. Is there something obvious I am missing? 

I have looked through the input file provided and I am a bit stuck with this. 

Kind regards, 
Zak

**Comment by davkovacs** [comment, marked as answer]
The `t` is the `--num_interactions` keyword in MACE. The default is 2, meaning two layers of MACE, and you can increase that.

**Reply by zakmachachi** [reply]
Thanks !
